frontend/src/components/ShoppingListDetail.tsx

```diff
@@ -50,16 +51,39 @@ export default function ShoppingListDetails() {
                 <h2>{list.listName}</h2>
                 {categories.map(category => (
                     <div key={category}>
-                        <h3>{category}</h3>
+                        <h3>{list.item.filter(item => item.category === category && !item.done)
```

Comment:
Da das "item" in list.item ein Array ist, wäre es besser Plural zu verwenden, also list.items